homeassistant/components/velux/manifest.json

```diff
@@ -1,10 +1,18 @@
 {
   "domain": "velux",
   "name": "Velux",
-  "codeowners": ["@Julius2342", "@DeerMaximum"],
+  "codeowners": ["@Julius2342", "@DeerMaximum", "@pawlizio"],
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/velux",
+  "integration_type": "hub",
   "iot_class": "local_polling",
+  "issue_tracker": "https://github.com/Julius2342/pyvlx/issues",
```

Comment:
The issue tracker field should not be added for built-in integrations as it is generated automatically
```suggestion
```